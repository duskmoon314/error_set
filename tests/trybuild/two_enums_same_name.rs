use error_set::error_set;

error_set!(
    SetLevelError {
        X {
        IoError2,
        },
        X {
        IoError,
        },
    }
);

pub fn main() {}