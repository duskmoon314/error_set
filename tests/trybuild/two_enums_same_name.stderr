error: 'X' already exists as an error enum.
  --> tests/trybuild/two_enums_same_name.rs:3:1
   |
3  | / error_set!(
4  | |     SetLevelError {
5  | |         X {
6  | |         IoError2,
...  |
11 | |     }
12 | | );
   | |_^
   |
   = note: this error originates in the macro `error_set` (in Nightly builds, run with -Z macro-backtrace for more info)
